### PR TITLE
dm(ui): support start/pause/resume/delete task

### PR DESCRIPTION
**Pull request by zoubingwu**
### What problem does this PR solve?

Add methods to start/pause/resume/delete task.

Issue Number: ref #3583

### What is changed and how it works?

Add buttons and event handlers

### Check List 

Tests 

 - Manual test (add detailed scripts or steps below)

### Release note 

```release-note
None
```

### CODE REVIEW

**Comment by zoubingwu** [comment]
cc @Ehco1996 @baurine

File: dm/ui/locales/zh.json [hunk 1]
```diff
@@ -93,9 +96,10 @@
   "source name": "名称",
   "source name is required": "名称不能为空",
   "start import": "开始导入",
+  "start task": "启动任务",
+  "start task and remove meta data": "清理元数据启动",
```
Comment (Ehco1996) [inline dm/ui/locales/zh.json:100]:
```suggestion
  "start task and remove meta data": "启动同步任务并清理元数据",
```

**Review by Ehco1996 — APPROVED** [review]
restLGTM

**Review by baurine — APPROVED** [review]

**Pull request merged**